is_core/generic_views/form_views.py

```diff
@@ -722,8 +754,12 @@ def get_fields(self):
 
     def get_fieldsets(self):
         return (
-            (self.core.bulk_change_fieldsets if hasattr(self.core, 'bulk_change_fieldsets') else None)
-            if self.fieldsets is None else self.fieldsets
+            (None, {'fields': self.get_fields()})
+        )
+
+    def generate_fieldsets(self, inline_views=None, **kwargs):
```

Comment:
for bulk change, is removed readonly and disallowed fields. Because show readonly fields in bulk change is nonsense